Solutions/Problem_Statement_57_Solution.java

```diff
@@ -0,0 +1,32 @@
+/*
+...
+Write a program to print the names of students by creating a Student class. If no name is passed
+while creating an object of Student class, then the name should be "Unknown", otherwise the name
+should be equal to the String value passed while creating object of Student class.
+....
+*/
+class Hack//creating a class hack
+{
+	public static void main(String[] args)
+	{
+		Student obj1=new Student("Riya");//created an object of Student class while passing parameter as name
```

Comment:
@yashasvi17rawat We get an output: 
Riya
Unknown
which is correct!
But I would like to have an user defined program. Could you please do the needful changes?